homeassistant/config.py

```diff
@@ -706,11 +706,14 @@ def _recursive_merge(
         if (not hass.config.skip_pip and p_integration.requirements and
                 not await async_process_requirements(
                     hass, p_integration.domain, p_integration.requirements)):
+            _LOGGER.error(
+                "Unable to install all requirements for %s.%s", domain, p_name)
             continue
 
         try:
             platform = p_integration.get_platform(domain)
         except ImportError:
+            _LOGGER.exception("Failed to get platform %s.%s", domain, p_name)
```

Comment:
This will be logged later during set up.

Reply:
Not with exception info. Ie you have no idea why.